VKApp/VKApp/Service/Operation/AsyncOperation.swift

```diff
@@ -0,0 +1,61 @@
+// AsyncOperation.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+
+/// AsyncOperation
```

Comment:
```suggestion
/// AsyncOperation
```
На ру или все коменты везде на англ?